src/Pages/yeonseonglee/Login/Login.js

```diff
@@ -0,0 +1,185 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: "",
+      pwdValue: "",
+    };
+  }
+
+  handleIdValue = (event) => {
+    this.setState({
+      idValue: event.target.value,
+    });
+  };
+
+  handlePwdValue = (event) => {
+    this.setState({
+      pwdValue: event.target.value,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push("/main-yeonseong");
+  };
+
+  render() {
+    const { idValue, pwdValue } = this.state;
+
+    return (
+      <div className="login">
+        <main className="wrapper">
+          <div className="wrapperLeft">
+            <img
+              className="leftimg"
+              src={"/images/yeonseonglee/instagram123.png"}
+              alt="insta"
+            />
+          </div>
+          <div className="wrapperRight">
+            <article className="artiTop">
+              <div className="logo">
+                <img src={"images/yeonseonglee/logo_text.png"} alt="logo" />
+              </div>
+              <div className="login">
+                <input
+                  className="idPwdForm"
+                  onChange={this.handleIdValue}
+                  type="text"
+                  placeholder="전화번호, 사용자이름 또는 이메일"
+                />
+                <input
+                  className="idPwdForm"
+                  onChange={this.handlePwdValue}
+                  type="password"
+                  placeholder="비밀번호"
+                />
+                <div>
+                  <button
+                    className={
+                      idValue.includes("@") && pwdValue.length > 5
+                        ? "loginButton active"
+                        : "loginButton inactive"
+                    }
```

Comment:
이 조건이 반복해서 쓰이니까 따로 변수로 빼서 쓸 수 있습니다
그리고 template literals 쓰면 더 줄일 수 있겠네요!
```suggestion
render(){
 return(
 const isValid =   idValue.includes('@') && pwdValue.length > 5;
 <button
  className={`loginButton ${isValid ? "active" : "inactive"}`}
```

Reply:
수정 완료 했습니다 !! : )